packages/types/src/package.ts

```diff
@@ -1,3 +1,12 @@
+export interface Fill {
+  slot: string;
```

Comment:
Can you add TSDocs to this file as well?

Reply:
I've tried to do this, but don't really see what benefit it brings to add the TSDoc here, to be honest 🤔 

TSDoc does not really have a way to "document" the interfaces. I also don't really know what I should document here? It's just a type for the `Fill`, it should be self-evident what it is.

Could you elaborate on what you had in mind?

Reply:
Let's not go over this, please. We need to add TSDocs to all our interfaces, functions, classes, types...

Interfaces are documented in the same way than classes.